examples/starrocks/starrocks-fe.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
   namespace: starrocks
 spec:
   starRocksFeSpec:
-    image: starrocks/fe-ubuntu:2.5.4
+    image: starrocks.gateway.scarf.sh/starrocks/fe-ubuntu:2.5.4
```

Comment:
as we talked, please change to `registry.starrocks.io`, and since we supports `latest` tag, 

we can change it to `registry.starrocks.io/fe-ubuntu:latest`